.tx/config

```diff
@@ -0,0 +1,9 @@
+[main]
+host = https://www.transifex.com
+lang_map = bg_BG: bg, cs_CZ: cs, fi_FI: fi, hu_HU: hu, nb_NO: nb, sk_SK: sk, th_TH: th, ja_JP: ja
```

Comment:
I don't know, but should't all languages be there? And if not, why?

Reply:
This is only the mapping of special ones to the general ones, where we don't have the general ones. So this is fine and used in all apps.